packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -76,10 +77,15 @@ def test_if_uiconfig_is_available(cluster):
 
 
 def test_if_dcos_history_service_is_up(cluster):
-    r = cluster.get('/dcos-history-service/ping')
 
-    assert r.status_code == 200
-    assert 'pong' == r.text
+    @retry(stop_max_delay=20000, wait_fixed=500)
```

Comment:
The purpose to modify this was to cover the flakiness in multi master scenarios. Ideally we would do what @mellenburg said and verify the service is up by checking last. We will create an issue to keep track of this and have it outside the scope of this commit.